### DISCUSSION TITLE
Video resume

### DISCUSSION THREAD

**Discussion by santhosh-tarento** [Sunbird-Knowlg/Community#165] [Bugs]
we are facing issue in video resume functionality where user left previously.  we are not able to set the current time or current position where video should play before video starts playing
https://codesandbox.io/s/video-js-7-lime-skin-forked-n66nsg?file=/index.js

**Comment by pallakartheekreddy** [comment]
@santhosh-tarento The information provided above is not clear, and it's challenging to understand the explanation. Can you please provide more details?

**Reply by santhosh-tarento** [reply]
sunbird video player used in sunbird-Ed-portal is not starting from where user left previously. suppose if he/she start the video and plays for some time and closes/logs out, after he/she comes back should be able to play the video where she/he left previously, currectly that is not happening.

As part of videojs library we need to set currentTime to set the position where video should play. however that is not working.


https://codesandbox.io/s/video-js-7-lime-skin-forked-n66nsg?file=/index.js.

**Reply by HarishGangula** [reply]
@santhosh-tarento  Can you share the input configuration given to the player when relaunching it to check this further.

**Reply by pallakartheekreddy** [reply]
@santhosh-tarento As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
